### DISCUSSION TITLE
Recombination hotspot

### DISCUSSION THREAD

**Discussion by marcustutert** [tskit-dev/msprime#1737] [Q&A]
I'm trying to figure out how to set the appropriate recombination rate within msprime.

My goal is to create a recombination hotspot 1000bp long in the middle of a 1Mb region (with no recombination elsewhere) where my effective population size is 5,000 and I want to generate population scaled recomb rates (4Ner) of 0,10 and 100.

I've worked through the simple maths and came up with a value of r = 0.05 (in the case of 4Ner = 1) I think that's correct (if not then let me know!) but in terms of using this value in msprime itself, what would be the best way to implement this (ratemap or something?) and do I need to scale this per Mb or do something else when I implement this within msprime as well?

**Comment by jeromekelleher** [comment]
Can you post what you've got so far @marcustutert, then maybe we can look and see if we think it does what you want?
